crates/bevy_window/src/system.rs

```diff
@@ -11,12 +12,41 @@ use bevy_input::{keyboard::KeyCode, Input};
 /// Ensure that you read the caveats documented on that field if doing so.
 ///
 /// [`WindowPlugin`]: crate::WindowPlugin
-pub fn exit_on_all_closed(mut app_exit_events: EventWriter<AppExit>, windows: Res<Windows>) {
+pub fn exit_on_all_closed(mut app_exit_events: EventWriter<AppExit>, windows: Query<&Window>) {
     if windows.iter().count() == 0 {
+        println!("no windows are open, exiting");
```

Comment:
This print should either be an `info!()` or removed

Reply:
```suggestion
```